### DISCUSSION TITLE
Default Field Value for reusable fields

### DISCUSSION THREAD

**Discussion by tibbis** [craftcms/cms#15807] [Ideas]
Just as the instructions/title for the reusable fields, it would be nice to have the default value as an override for reusable text/number fields.

**Comment by BigglesZX** [comment]
I was also wondering about this today. There may be good reasons why it's not possible, but I'd be interested to know if it is. Likewise for Dropdown fields.

**Comment by alexanderbuergin** [comment]
1+

**Comment by foxtrotcz** [comment]
This sounds similar to this past request? https://github.com/craftcms/cms/discussions/8688
